website/docs/docs/collaborate/manage-access/set-up-snowflake-oauth.md

```diff
@@ -41,7 +41,7 @@ CREATE OR REPLACE SECURITY INTEGRATION DBT_CLOUD
 | ENABLED  | Required |
 | OAUTH_CLIENT  | Required |
 | OAUTH_CLIENT_TYPE  | Required |
-| OAUTH_REDIRECT_URI  | Required. If dbt Cloud is deployed on-premises, use the domain name of your application instead of `cloud.getdbt.com` |
+| OAUTH_REDIRECT_URI  | Required. If dbt Cloud is deployed on-premises, use the domain name of your application instead of `cloud.getdbt.com`. If you're on the EMEA servers, use `emea.dbt.com` for the redirect URI. |
```

Comment:
```suggestion
| OAUTH_REDIRECT_URI  | Required. Use the access URL that corresponds to your server [region](/docs/deploy/regions). If dbt Cloud is deployed on-premises, use the domain name of your application instead of the access URL. |
```

Reply:
suggestion small tweak based on Leona's feedback

Reply:
Perfect!  Thank you.

Reply:
Perfect!  Thank you @mirnawong1 and @runleonarun